test/integration/base.py

```diff
@@ -322,7 +327,7 @@ def setUp(self):
                 'test_original_source_path={0.test_original_source_path}',
                 'test_root_dir={0.test_root_dir}'
             )).format(self)
-            logger.exception(msg)
+            fire_event(IntegrationTestSetupFailure(msg=msg))
```

Comment:
Ok so I'm actually tempted to make an escape hatch event for tests. Like one class we just pass strings to since they will only ever be used for debugging tests.

If we want test messages to be typed like this, I think I want them to be in a separate types module within tests...

What do you think?

Reply:
@iknox-fa I'd like your thoughts too.

Reply:
I lean towards the escape hatch class.  Right now these are the only 4 logs generated in any tests.  The escape hatch makes it easy to add more logs when testing if needed without changing code elsewhere to add new types, which i like.

Reply:
Yeah, that makes sense to me, especially putting it in a separate module for separation of concerns.

Reply:
With current logging using different levels I ended up creating a new module and standard messages to go with it.